lib/babel-config.js

```diff
@@ -14,9 +12,9 @@ function validate(conf) {
 	const isValidShortcut = conf === 'default' || conf === 'inherit';
 
 	if (!conf || (typeof conf === 'string' && !isValidShortcut)) {
-		let message = colors.error(figures.cross);
+		let message = figures.cross;
 		message += ' Unexpected Babel configuration for AVA. ';
-		message += 'See ' + chalk.underline('https://github.com/avajs/ava#es2015-support') + ' for allowed values.';
+		message += 'See https://github.com/avajs/ava#es2015-support for allowed values.';
```

Comment:
Oh, I didn't notice this removes it from the actual code. In https://github.com/avajs/ava/pull/1198#discussion_r98352347, I was talking about removing the chalk call from the test, not elsewhere.

> Is this because execCli spawns a new process and the chalk.enabled and colors.enabled stuff is not passed on?

Not exactly. Colors used to be forced "on", now it's auto-detected and colors are automatically off when spawned.